indico/web/client/js/react/components/index.js

```diff
@@ -32,3 +32,4 @@ export {default as ResponsivePopup} from './ResponsivePopup';
 export {default as PublicationSwitch} from './PublicationSwitch';
 export {default as MathJax} from './MathJax';
 export {default as FileSubmission} from './FileSubmission';
+export {default as EditableSubmissionButton} from './EditableSubmissionButton';
```

Comment:
I think this component would fit better in the editing module than here, since this place is meant for common components not specific to some module.